Testing/Unit/Python/dummy_viewer.py

```diff
@@ -0,0 +1,15 @@
+#! /usr/bin/env python3
+
+import sys
+import os
+
+
+out = ""
+
+if len(sys.argv) > 1:
+    for x in sys.argv[1:]:
+        out = out + ' ' + x
+
+
+print(out)
+os.environ['DUMMY_SHOW'] = x

```

Comment:
The variable 'x' doesn't exist till assigned a value in the loop. If no argument is given this causes an exception. Was the intent to use the 'out' variable and not the 'x' variable?